node/full.go

```diff
@@ -51,6 +53,25 @@ const (
 
 var _ Node = &FullNode{}
 
+var VerifierFn = func(proxyApp proxy.AppConns) func(fraudProof fraud.Proof) (bool, error) {
```

Comment:
[follow-up] We might want to move `var VerifierFn` to somewhere common, maybe ProofService?

Reply:
moved to proof service factory.